### PR TITLE
[LA-2023] Updating Los Angeles website

### PR DESCRIPTION
**Pull request by omargohan**
**Summary of changes:**

- Adding nav items
- Adding images
- Adding sponsor information
- Adding CFP link

**Tested:**

- Locally using Hugo container

### CODE REVIEW

File: content/events/2023-los-angeles/sponsor.md [hunk 1]
```diff
@@ -1,66 +1,43 @@
...
 +++
+<br>
+<h3>Become A Sponsor</h3>
 
-We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+<p>
+Thank you for your interest! Below is an overview of the conference as well as our sponsor packages. Please email the organizing committee at {{< email_organizers >}} for additional information or to secure your package.
 
-<hr>
+<h3>About DevOpsDayLA</h3>
```
Comment (jyee) [inline content/events/2023-los-angeles/sponsor.md:12]: There's a few spots where this is DevOpsDayLA and others in the text where it's DevOpsDaysLA (day vs days). This should probably be standardized for consistency. I think it's been singular in the past for the 1-day event, but I don't think anyone on core has a strong feeling on this one way or the other.

**Review by jyee — COMMENTED** [review]
The other thing you may want to do is update the `propose.md` file. That's still linked from the "Propose" button in the header of the event. If I recall, it's harder to edit that one (or at least I dont recall where/how), so updating the propose page to point to the SCALE cfp page is a good work around.

File: content/events/2023-los-angeles/sponsor.md (hunk 1 above)
Comment (irabinovitch) [reply content/events/2023-los-angeles/sponsor.md:12]: Agreed. @omargohan can we fix in a follow up PR?

File: content/events/2023-los-angeles/sponsor.md [hunk 2]
```diff
@@ -1,66 +1,43 @@
...
 <p>
-Gold sponsors get a full table and Silver sponsors a shared table where they can interact with those interested to come visit during breaks. All attendees are welcome to propose any subject they want during the open spaces, but this is a community-focused conference, so heavy marketing will probably work against you when trying to make a good impression on the attendees.
+DevOpsDaysLA, now in its 10th year, is Southern California's only annual conference focused on the practice of DevOps. The conference provides a relaxed, casual, and diverse environment where sponsors connect with talent, demonstrate the latest tools, and offer world class services to facilitate organizations' DevOps journey.
```
Comment (irabinovitch) [inline content/events/2023-los-angeles/sponsor.md:15]: i think its 11 years now

File: content/events/2023-los-angeles/sponsor.md (hunk 1 above)
Comment (omargohan) [reply content/events/2023-los-angeles/sponsor.md:12]: Sure, I will fix that.

File: content/events/2023-los-angeles/sponsor.md (hunk 2 above)
Comment (omargohan) [reply content/events/2023-los-angeles/sponsor.md:15]: Got it,

**Comment by irabinovitch** [comment]
@jyee I think Omar has addressed all the feedback.

File: content/events/2023-los-angeles/welcome.md [hunk 3]
```diff
@@ -1,47 +1,42 @@
...
+</div>
 
 <!-- <div class = "row">
   <div class = "col-md-2">
-    <strong>Propose</strong>
+    <strong>Call For Papers</strong>
   </div>
   <div class = "col-md-8">
-    {{< event_link page="propose" text="Propose a talk!" >}}
+    <a href="https://docs.google.com/forms/d/e/1FAIpQLSewxucXuZ5nmZYX-nBitrmnr1ZA-O7oc33kheNvoRw1ufEbgg/viewform">
```
Comment (nathenharvey) [inline content/events/2023-los-angeles/welcome.md:37]: Should this be linking to https://www.socallinuxexpo.org/scale/20x/cfp instead of the Google form?

Comment (irabinovitch) [reply content/events/2023-los-angeles/welcome.md:37]: Yes.  Doesn't look like my PR got merged on Omar's branch. That being said if we can get this one merged overall Ill follow up with another PR to fix the CFP link and any other feedback shortly there after.

File: content/events/2023-los-angeles/sponsor.md (hunk 2 above)
Comment (brynak) [reply content/events/2023-los-angeles/sponsor.md]: 2023 is the 12th year. It's correct in the sponsor prospectus.

File: content/events/2023-los-angeles/welcome.md (hunk 3 above)
Comment (yvovandoorn) [reply content/events/2023-los-angeles/welcome.md:37]: I'd suggest fixing this in the same PR instead of a follow up.

Comment (irabinovitch) [reply content/events/2023-los-angeles/welcome.md:37]: I have no idea when Omar will have time to update his PR or merge the changes I PR'd on his branch. If you just merge his changes I'll have a new PR for you with in a few minutes.

Comment (irabinovitch) [reply content/events/2023-los-angeles/welcome.md:37]: At least we have access to the form to see submissions, which while not ideal works. Where as anyone who is sending out proposals based on the current version of the website is having their proposals go to /dev/null. With the CFP closing in 2 weeks we'd really like to get the site updated ASAP with more accurate sponsorship and CFP information.

Comment (omargohan) [reply content/events/2023-los-angeles/welcome.md:37]: @yvovandoorn that piece of code is commented out. I tested every part of the feedback we received from you and, as Ilan said, we addressed everything in this PR. Could you please test it using Hugo? This link that you are referring to it is not in being used. Thank you.

**Comment by brynak** [comment]
Thank you for your assistance and QA efforts. 

If there are no functionality issues remaining with this PR, please approve this without delay. 

Any content issues can be fixed in the next PR.

Ship fast. Ship often. Free the PR.

Love, 
Bryna

**Pull request merged**